File: Frontend/barbershop/src/types/ResponseDTOs/brandDTO.ts
import type { ProductDTO } from './productDTO';

export type BrandDTO = {
    brandId: number;
    brandName: string;
    description?: string;
    logoPath?: string;
    isActive?: boolean;
    createdAt?: string; // ISO string
    updatedAt?: string; // ISO string
    products?: ProductDTO[];
};

File: Backend/barbershop/barbershop/Models/ResponseDTOs/BrandDTO.cs
using barbershop.Models.Entitys;

namespace barbershop.Models.ResponseDTOs
{
    public class BrandDTO
    {
        public int BrandId { get; set; }

        public string BrandName { get; set; } = null!;

        public string? Description { get; set; }

        public string? LogoPath { get; set; }

        public bool? IsActive { get; set; }

        public DateTime? CreatedAt { get; set; }

        public DateTime? UpdatedAt { get; set; }

        public List<ProductDTO>? ProductDTO { get; set; } = new List<ProductDTO>();
    }
}
